src/windows-connect.js

```diff
@@ -31,7 +34,7 @@ function connectToWifi(config, ap, callback) {
       }
 
       fs.writeFileSync(
-        'nodeWifiConnect.xml',
+        path.join(tempDir, 'nodeWifiConnect.xml'),
```

Comment:
I propose you to create a variable outside the function so that we are sure the filename is always the same. Something like that : 

```js
const profileFilename = path.join(tempDir, 'nodeWifiConnect.xml');
```

So that you use this variable after in the code.

Reply:
Thanks @friedrith for the quick review. I have updated the pull request by incorporating the suggested change.